### PR TITLE
Support Python 3.10-rc.1

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def get_packages(package):
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```
Comment (graingert) [inline setup.py:91]:
```suggestion
```
we can't add the trove until https://github.com/aaugustin/websockets/issues/1000 is on pypi

Comment (Kludex) [reply setup.py:91]: But they have added as well https://github.com/aaugustin/websockets/blob/6edb363af07365867b4c372405b6ab3177a57830/setup.py#L53

Comment (Kludex) [reply setup.py:91]: Ah, it's on master.

Comment (Kludex) [reply setup.py:91]: But `websockets` is not a mandatory dependency, so it shouldn't be a blocker, right?

**Review by graingert — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks @graingert !

I guess you'll bump the websockets on a new PR?

**Pull request merged**